gateway/dapiimpl/uatracing.go

```diff
@@ -0,0 +1,26 @@
+package dapiimpl
+
+import (
+	"context"
+	"net/http"
+
+	"go.opentelemetry.io/otel/propagation"
+
+	"github.com/oapi-codegen/runtime/strictmiddleware/nethttp"
+	"go.opentelemetry.io/otel"
+)
+
+func NewUserAgentTracingHandler() func(f nethttp.StrictHTTPHandlerFunc, operationID string) nethttp.StrictHTTPHandlerFunc {
```

Comment:
This should probably be called something like NewOtelTracingHandler rather than UserAgentTracingHandler.

Reply:
Done